tests/test_backend.py

```diff
@@ -2353,6 +2376,9 @@ def func(x):
             return tf.identity(x_, name=_TFOUTPUT)
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val})
 
+    @skip_tflite("skip_tflite is not support unit32 when tf version <= 2.3.0")
```

Comment:
... does not support unit32 if tf version <= 2.3.0.

Reply:
Fixed!